staking/types/validator.go

```diff
@@ -331,7 +331,8 @@ func (w *ValidatorWrapper) SanityCheck(
 			w.Delegations[0].Amount.Cmp(w.Validator.MinSelfDelegation) < 0 {
 			return errors.Wrapf(
 				errInvalidSelfDelegation,
-				"have %s want %s", w.Delegations[0].Amount.String(), w.Validator.MinSelfDelegation,
+				"min_self_delegation %s, after undelegation amount %s",
```

Comment:
message should not say `undelegation` right? as `SanityCheck` is also called in other staking messages like Delegate, Create, Edit, and CollectReward. May be just print min_self_delegation, and amount.

Reply:
Changed.